test/edu-benefits/1990/schema/schema.unit.spec.js

```diff
@@ -1,6 +1,6 @@
 import { expect } from 'chai';
 import Ajv from 'ajv';
-import { eduBenefits as schema } from 'vets-json-schema';
```

Comment:
I'm not sure how it was passing before either, or why this would cause an issue. Tests pass also pass locally for me. Going to try to restart the build and see if it was just a fluke (which seems to have happened before).